src/newsroom-context/hooks/useGetTranslationUrl.ts

```diff
@@ -9,25 +9,26 @@ import { useCurrentCategory } from './useCurrentCategory';
 import { useCurrentStory } from './useCurrentStory';
 
 // Pulled from SDK types to not leak the `@prezly/sdk` dependency into the client bundle
-enum StoryVisibility {
+// It looks like a hack, but the name of enum must be the same as in SDK
+enum Visibility {
```

Comment:
This is tricky, string literal types would fit here better

Reply:
Yeah, this whole part is a bit of a hack, but otherwise Next pulls the whole SDK dependency right into the client bundle 😕

Reply:
I thought that Next.js should be smart enough to do tree-shaking. Or is it the case for ESM dependencies only?

Reply:
It should do tree-shaking for ESM, but currently theme-kit is CJS, and experimental move to ESM wasn't successful. Next has some really weird issues when there is a combination of CJS and ESM modules in the deps.